mitodl/odl-video-service
# Make the repo more standardized with webpack etc.

#### What are the relevant tickets?
Fixes https://github.com/mitodl/odl-video-service/issues/26

#### What's this PR do?
Adds many standard files created by cookiecutter, primarily for JS/React development and CSS.

#### How should this be manually tested?
Run the webpack script and test_suite script, there should be no errors.
```./webpack_dev_server.sh --install```
```./scripts/test/test_suite.sh```
